proxy/http/HttpConfig.h

```diff
@@ -824,6 +824,8 @@ struct HttpConfigParams : public ConfigInfo {
 
   MgmtInt body_factory_response_max_size;
 
+  MgmtByte forward_proxy_ftp_enabled;
```

Comment:
Can you move this up where other MgmtByte's are? That avoids unnecessary padding of the bytes (as it is above, you would waste at least 3 bytes in the struct).